src/logic/proposal-logic.ts

```diff
@@ -2,7 +2,12 @@ import useActions from '@hooks/useActions'
 import { useGardenState } from '@providers/GardenState'
 import { useProposal } from '@hooks/useProposals'
 
-export default function useProposalLogic(match) {
+export default function useProposalLogic(match: {
```

Comment:
Why not export it to type struct? Some special reason?

Reply:
No special reason, when its a small type I don't mind adding directly. But for coherence purpose I can updated.